views/components/LoadingScreen.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import GameScene from '../components/GameScene';
+const ReactTHREE = require('react-three');
+const THREE = require('three');
+const Mesh = ReactTHREE.Mesh;
+
+class LoadingScreen extends React.Component {
+  constructor() {
+    super();
+    this.font = null;
+    this.loadFont()
+  }
+
+  loadFont(){
+    var loader = new THREE.XHRLoader(THREE.DefaultLoadingManager)
+    loader.load('fonts/helvetiker_regular.typeface.json', ( font )=>{
+      this.font = new THREE.Font(JSON.parse(font));
```

Comment:
This should be moved to a separate action, and also you need to `try...catch` `JSON.parse`